js/components/ArbeidsrettetOppfolging.js

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import { getHtmlLedetekst, getLedetekst, Utvidbar } from 'digisyfo-npm';
+import Veilederpanel from 'nav-frontend-veilederpanel';
+import { Hovedknapp, Knapp } from 'nav-frontend-knapper';
+import history from '../history';
+import Sidetopp from './Sidetopp';
+
+const ArbeidsrettetOppfolging = () => {
+    return (
+        <React.Fragment>
+            <Sidetopp
+                tittel={getLedetekst('infoside-fo.sidetittel')}
+            />
+            <div className="rad rad--graa">
```

Comment:
Kunne kanskje putta divene inn i sitt tilhørende `<Rad />` element? Da kan man kanskje spare seg for en ekstra div. Blir også litt mer lesbart 👓